components/Nav/Nav.tsx

```diff
@@ -102,6 +102,15 @@ export default function Nav({
               Organizations
             </li>
           </Link>
+          <Link href="/voting-guides" passHref>
+            <li
+              className={`${styles.navItem} ${
+                pathname.includes("/voting-guides") && styles.active
+              }`}
```

Comment:
I like using https://github.com/JedWatson/classnames#usage for this kinda conditional application of classes. Definitely find it easier to manipulate an object vs adding interpolated strings in the rendered HTML.

Reply:
yea thanks for pulling that in @neko-mamushi totally agree.  lets stick with the `classnames` approach it makes the css module setup much more usable

Reply:
@neko-mamushi would you mind providing the example of how you would use classnames here?

Reply:
I dig classnames! I saw this done somewhere else in the code, but saw Sharan was using classnames in some more complex cases. Still figuring out what our conventions are, but no arguments there, classnames is great.

Reply:
The rest of the file is done the same way... should we convert the whole file to classnames? Perhaps this should be done as a separate task?